src/test/run-make-fulldeps/issue-26092/Makefile

```diff
@@ -1,4 +0,0 @@
--include ../tools.mk
-
-all:
-	$(RUSTC) -o "" blank.rs 2>&1 | $(CGREP) -i 'No such file or directory'
```

Comment:
Could you keep the test, but just check that rustc does not panic instead of checking for this very message?

Reply:
I tried to do that by https://github.com/rust-lang/rust/pull/96544/commits/1147d50050788851438e80c57279b99b9a457924 . I confirmed that this test works well by intentionally invoking a panic. However, is there any smarter way to ensure the same thing?